datacleaner/datacleaner.py

```diff
@@ -72,11 +83,18 @@ def autoclean(input_dataframe, drop_nans=False, copy=False):
 
         # Encode all strings with numerical equivalents
         if str(input_dataframe[column].values.dtype) == 'object':
-            input_dataframe[column] = LabelEncoder().fit_transform(input_dataframe[column].values)
+            if encoder is None:
+                input_dataframe[column] = encoder().fit_transform(input_dataframe[column].values)
```

Comment:
We should perform all of the encoder operations here on a column-by-column basis instead of down at line 92. From my tests, sklearn preprocessors can't encode multiple columns at once.